packages/serverless-deploy-iam/bin/app.ts

```diff
@@ -294,6 +294,88 @@ export class ServiceDeployIAM extends cdk.Stack {
+                              "cognito-sync:SetIdentityPoolConfiguration",
+                              "cognito-sync:SubscribeToDataset",
+                              "cognito-sync:UnsubscribeFromDataset",
+                              "cognito-sync:UpdateRecords",
+                              "cognito-identity:CreateIdentityPool",
+                              "cognito-identity:DeleteIdentities",
+                              "cognito-identity:DeleteIdentityPool",
+                              "cognito-identity:DescribeIdentity",
+                              "cognito-identity:DescribeIdentityPool",
+                              "cognito-identity:GetCredentialsForIdentity",
+                              "cognito-identity:GetId",
+                              "cognito-identity:GetIdentityPoolRoles",
+                              "cognito-identity:GetOpenIdToken",
+                              "cognito-identity:GetOpenIdTokenForDeveloperIdentity",
+                              "cognito-identity:GetPrincipalTagAttributeMap",
+                              "cognito-identity:ListIdentities",
+                              "cognito-identity:ListIdentityPools",
+                              "cognito-identity:ListTagsForResource",
+                              "cognito-identity:LookupDeveloperIdentity",
+                              "cognito-identity:MergeDeveloperIdentities",
+                              "cognito-identity:SetIdentityPoolRoles",
+                              "cognito-identity:SetPrincipalTagAttributeMap",
+                              "cognito-identity:TagResource",
+                              "cognito-identity:UnlinkDeveloperIdentity",
+                              "cognito-identity:UnlinkIdentity",
+                              "cognito-identity:UntagResource",
+                              "cognito-identity:UpdateIdentityPool",
+                         ]
+                    },
+                    {
+                         name: 'COGNITO_IDP',
+                         prefix: `arn:aws:cognito-idp:${region}:${accountId}:userpool`,
+                         qualifiers: [`${serviceName}*`, `${region}_*`],
+                         actions: [
+                              "cognito-idp:Create*",
+                              "cognito-idp:Delete*",
+                              "cognito-idp:Describe*",
+                              "cognito-idp:Get*",
+                              "cognito-idp:List*",
+                              "cognito-idp:Set*",
+                              "cognito-idp:TagResource",
+                              "cognito-idp:UntagResource",
+                              "cognito-idp:Update*",
+                         ]
+                    },
+                    {
+                         name: 'COGNITO_IDP_CREATEUSERPOOL',
+                         prefix: `arn:aws:cognito-idp:${region}:${accountId}:userpool`,
+                         qualifiers: ["*"],
```

Comment:
Instead of using `qualifiers` and `prefix` here you can optionally use `resources`. See `API_GATEWAY` for an example. Both achieve the same thing though!